easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,49 @@
+#//////////////////////////////////////////////////////////////////////////////
+#   -- MAGMA (version 1.6.1) --
+#      Univ. of Tennessee, Knoxville
+#      Univ. of California, Berkeley
+#      Univ. of Colorado, Denver
+#      @date January 2015
+#//////////////////////////////////////////////////////////////////////////////
+
+# GPU_TARGET contains one or more of Tesla, Fermi, or Kepler,
+# to specify for which GPUs you want to compile MAGMA:
+#     Tesla  - NVIDIA compute capability 1.x cards (no longer supported in CUDA 6.5)
+#     Fermi  - NVIDIA compute capability 2.x cards
+#     Kepler - NVIDIA compute capability 3.x cards
+# The default is "Fermi Kepler".
+# See http://developer.nvidia.com/cuda-gpus
+#
+#GPU_TARGET ?= Fermi Kepler
+GPU_TARGET = Kepler
+
+CXX       = CC
```

Comment:
relevant recent bug for OpenFOAM: https://github.com/hpcugent/easybuild-easyconfigs/issues/1842